data/family/Debian.yaml

```diff
@@ -0,0 +1 @@
+xymon::client::include_clientlaunch_d: false
```

Comment:
Why did you add this line as the parameter defaults to false? Just for visibility?

Reply:
well, eh, maybe. You are right, they are obsolete, will remove them